android/README.md

```diff
@@ -0,0 +1,218 @@
+# Android RUM Instrumentation Workshop
+
+This workshop is intended to lead you through the process of taking an existing Android app and adding Splunk RUM
+instrumentation to it.
+
+## Prerequisites
+
+This workshop assumes that you have done the following:
+
+* Have `git` installed and can use it to clone open source repositories (such as this one).
+* Installed [Android Studio](https://developer.android.com/studio/)
+* Gone through the first 2 lessons for Android developers:
+    - "Create an Android Project"  https://developer.android.com/training/basics/firstapp/creating-project
+    - "Run Your App"  https://developer.android.com/training/basics/firstapp/running-app
+* Have access to a Splunk RUM enabled organization in the Splunk Observability product.
+* Have a valid accessToken with the `RUM` authorization scope attached.
```

Comment:
you can also add a link here so that people know how to create one: https://docs.splunk.com/Observability/rum/rum.html#step-1-generate-your-rum-token-in-the-observability-cloud